anthos-bm-gcp-bash/FAQ.md

```diff
@@ -0,0 +1,51 @@
+# Frequently Asked Questions (FAQ)
+
+The following are a list of variations with which the script can be used to
+deploy Anthos on bare metal in Compute Engine VMs.
+
+---
+
+### Using a network other than the `default`
+
+If you want to use a network other than the `default` network then you will have
+to do the following changes:
+
+1. Ensure that `TCP`, `UDP` and `SSH` traffic is allowed on the network of your
+   choice.
+   ```sh
+   gcloud compute firewall-rules create abm-allow-internal \
+        --project=$PROJECT_ID \
+        --network=<YOUR_VPC_NETWORK> \
+        --direction=INGRESS \
+        --action=ALLOW \
+        --rules=tcp:0-65535,udp:0-65535,icmp \
+        --source-ranges=10.128.0.0/9 \
+        --priority=65534
+   ```
+   ```sh
+   gcloud compute firewall-rules create abm-allow-ssh \
+        --project=$PROJECT_ID \
+        --network=<YOUR_VPC_NETWORK> \
+        --direction=INGRESS \
+        --action=ALLOW \
+        --rules=tcp:22 \
+        --source-ranges=0.0.0.0/0 \
```

Comment:
Note that this is dangerous and allows _all_ incoming traffic. Is that intentional?

Reply:
This is intended. This is a getting started guide which is expected to run in the `default` network which has open `firewall rules`. So the variations are added to just show case what to do to replicate the `default` network to get the guide working. But the user may decide to change the commands as they wish

Reply:
Added a note about it - just in case: 282835437a4ca6986f1d38cd2f626d929024d0b3